proxy/logging/LogFile.cc

```diff
@@ -255,15 +255,15 @@ LogFile::close_file()
       ::close(m_fd);
```

Comment:
Shouldn't we be checking the return value of close?  If we aren't really closing a file, returned an error, then should we be decrementing the number of open files?

Reply:
Sounds good. I can add that.